go/private/context.bzl

```diff
@@ -697,10 +697,12 @@ def _cgo_context_data_impl(ctx):
     )
 
     return [CgoContextInfo(
-        crosstool = find_cpp_toolchain(ctx).all_files.to_list(),
+        crosstool = cc_toolchain.all_files.to_list(),
         tags = tags,
         env = env,
         cgo_tools = struct(
+            cc_toolchain = cc_toolchain,
+            feature_configuration = feature_configuration,
```

Comment:
Is it safe to save this object across contexts? I'm not sure.

Reply:
I'm not sure how it is possible to make multiple contexts?

Reply:
What I mean is that `feature_configuration` might be specific to the context where it was obtained. It may not be safe to save the object obtained when analyzing `//:cgo_context_data` and use that when analyzing other targets.

I actually don't know though. I would guess it's safe to just grab a new `feature_configuration` when it's needed.

Reply:
Considering the whole toolchain and actions are resolved there, I think it's best to keep the features as well, or else we risk mixing them at a later safe (`c_compile_options` etc...)